students/meet_shah/lesson09/donor_class.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+from functools import reduce
+
+class Donor:
+
+    def __init__(self, name, donations=None):
+        self._name = name
+        self._donations = [] if donations is None else [donations]
+
+
+    @property
+    def name(self):
+        return self._name
+
+
+    @property
+    def donations(self):
+        return self._donations
+
+
+    def add_donations(self, donation):
```

Comment:
careful with naming -- this only adds a single donation -- better to call it "add_donation"